app/helpers/result_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ResultHelper
+  def title_string(object, kind)
+    invalid_date = kind == 'conviction' ? 'never' : 'caution_result'
```

Comment:
I think if we get rid of this statement, the code will be simpler and have the same result. Naming the locale key to `no_date` will also be more explicit.

In the following line we can then just say:

`return t("title.no_date", scope: "results/#{kind}") unless object.instance_of?(Date)`